src/main/java/me/ryanhamshire/GriefPrevention/Claim.java

```diff
@@ -597,15 +597,15 @@ public String allowGrantPermission(Player player)
         if (this.allowEdit(player) == null) return null;
 
         //anyone who's in the managers (/PermissionTrust) list can do this
-        for (int i = 0; i < this.managers.size(); i++)
+        for (String managerID : this.managers)
         {
-            String managerID = this.managers.get(i);
+            if (managerID == null) continue;
             if (player.getUniqueId().toString().equals(managerID)) return null;
 
             else if (managerID.startsWith("[") && managerID.endsWith("]"))
             {
                 managerID = managerID.substring(1, managerID.length() - 1);
-                if (managerID == null || managerID.isEmpty()) continue;
+                if (managerID.isEmpty()) continue;
```

Comment:
I didn't re-add the useless null check that I moved up to L602 (which was also supposed to be in that second commit, my bad) but I've otherwise reverted the change here.

Reply:
For clarity, the null check was valid defensive coding, but in its old position it didn't actually do anything because String#substring never returns null. It is, however, possible for bad addons to add null managers.